src/Tribe/Commerce/PayPal/Main.php

```diff
@@ -2991,23 +3045,26 @@ public function get_attendee( $attendee, $post_id = 0 ) {
 			'qr_ticket_id'  => $attendee->ID,
 			'security_code' => $security,
 
-			// Attendee Meta
+			// Attendee Meta.
 			'attendee_meta' => $meta,
-		) );
+
+			// Handle initial Attendee flags.
+			'is_subscribed' => tribe_is_truthy( get_post_meta( $attendee->ID, $this->attendee_subscribed, true ) ),
+			'is_purchaser'  => true,
+		] );
```

Comment:
:no_entry_sign: Closing parenthesis of a multi-line function call must be on a line by itself

<sub>hash: eb5224e81cbcf1c8ecd0c6e4418e47f5</sub>